core/src/kvs/surrealcs/mod.rs

```diff
@@ -72,8 +72,10 @@ impl Drop for Transaction {
 impl Datastore {
 	/// Open a new database
 	pub(crate) async fn new(path: &str) -> Result<Datastore, Error> {
-		create_connection_pool(path, None).await.unwrap();
-		Ok(Datastore {})
+		match create_connection_pool(path, None).await {
+			Ok(_) => Ok(Datastore {}),
+			Err(_) => Err(Error::Ds("error connecting to surrealcs".to_string())),
```

Comment:
Would it be useful to also show a reason for the connection error?

Reply:
from what I understand this is a `SurrealCS` issue and normal users should not be using it. @tobiemh do you want the connection error to be in the error as well as the note that there's a problem connecting to SurrealCS?